### PR TITLE
Destructable Street Lamps

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

File: Assets/_PartyParkade/Script/Game/Destructables/StreetLampBehaviour.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                {
+                    m_particleSystem.Stop();
+                }
+            }
+        }
+
+        private void StreetLampUpdate()
+        {
+            m_timeBeforeStopParticleEffects -= Time.deltaTime;
+            Debug.Log("Update time" + m_timeBeforeStopParticleEffects);
```
Comment (wbeltran777) [inline Assets/_PartyParkade/Script/Game/Destructables/StreetLampBehaviour.cs]: Remove debug log

File: Assets/_PartyParkade/Script/Game/Destructables/StreetLampBehaviour.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            /// I set this up specifically for lamps since it can't stand up and stay still by itself
+            m_rigidbody.isKinematic = true;
+            transform.localPosition = Vector3.zero;
+            RandomizeTimeBeforeStopParticleEffect();
+        }
+
+        private void RandomizeTimeBeforeStopParticleEffect()
+        {
+            m_timeBeforeStopParticleEffects = Random.Range(m_minMaxAliveTime.x, m_minMaxAliveTime.y);
+            Debug.Log(m_timeBeforeStopParticleEffects);
```
Comment (wbeltran777) [inline Assets/_PartyParkade/Script/Game/Destructables/StreetLampBehaviour.cs]: Remove debug log

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Game/Destructables/StreetLampBehaviour.cs]: Removed

File: Assets/_PartyParkade/Script/Game/Destructables/StreetLampBehaviour.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/_PartyParkade/Script/Game/Destructables/StreetLampBehaviour.cs]: Removed

**Comment by mikeslgoh** [comment]
[Investigate breakable objects](https://trello.com/c/g1kCCKqD/507-investigate-breakable-objects)

**Pull request merged**